packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -1,3 +1,14 @@
+// saveResponseToDB seems to be the only thing needing factorisation here.
+// I haven't found the code that instantiates this class yet but potentially
+// the api patch function to called could be passed into this?
+
+// Having searched the codebase from "new Question" to no avail, I seem to
+// recall there's some bizarre programmatic instantiation going on somewhere.
```

Comment:
I was thinking of Journeys, but that doesn't happen anymore. I think this class is just "abstract" and the other questions extend it